### PR TITLE
[WIP][8.0][BR000973][T22092]Refactor elico_project into several distinct modules[project_task_work_bottom_edit]

### PR DESCRIPTION
**Pull request by dongshukai**
This PR is for module project_task_work_bottom_edit

### CODE REVIEW

File: project_task_work_bottom_edit/README.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+To install this module, you need to:
+
+ * have basic modules installed (project)
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline project_task_work_bottom_edit/README.rst]: check https://github.com/Elico-Corp/odoo-addons/pull/171/files

File: project_task_work_bottom_edit/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Task Work Bottom Edit',
+    'version': '8.0.1.0.1',
```
Comment (elicoidal) [inline project_task_work_bottom_edit/__openerp__.py]: 8.0.1.0.0

**Review by elicoidal — CHANGES_REQUESTED** [review]
details

**Comment by dongshukai** [comment]
@Elico-Corp/odoo-reviewers 
pls check

**Review by elicoidal — APPROVED** [review]

**Pull request merged**